src/Microsoft.Framework.PackageManager/Restore/NuGet/NuGetv2Feed.cs

```diff
@@ -170,6 +175,16 @@ public PackageInfo BuildModel(string id, XElement element)
             var idElement = properties.Element(_xnameId);
             var titleElement = element.Element(_xnameTitle);
 
+            var publishElement = properties.Element(_xnamePublish);
+            if (publishElement != null)
+            {
+                DateTime publishDate; 
+                if (DateTime.TryParse(publishElement.Value, out publishDate) && (publishDate == _unlistedPublishedTime))
+                {
+                    return null; 
```

Comment:
I'm not sure about changing the `PackageInfo` interface. The `IsPublish` value maybe useful, but we don't know yet. We can get back to save the value once we have a case.